.prow.yaml

```diff
@@ -147,6 +147,39 @@ presubmits:
           resources:
             requests:
               cpu: 1
+  - name: pull-kubeone-e2e-aws-flatcar-containerd-conformance-1.18
```

Comment:
I'm not sure should we run Flatcar E2E tests on each PR. Currently, we're testing Flatcar periodically (each workday in the morning), and that seems to be enough. Historically, Flatcar tests were flakier than Ubuntu ones, although, they seem much more stable nowadays.

Should we move this test to periodic along with other Flatcar E2E tests?

@kron4eg WDYT?

Reply:
I'd keep it, for that we could spontaneously verify that it's still working when we might have doubts.